app/models/patient.rb

```diff
@@ -129,6 +129,136 @@ def latest_transfer
       .where('latest_assessment_at < ? OR latest_assessment_at IS NULL', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)
   }
 
+  # Patients who are eligible for reminders
+  #
+  # GENERAL SCOPE OVERVIEW:
+  # - Everything is before the OR is essentially checking if the HoH or the patient
+  #   who isn't in a household is eligible on their own.
+  # - Everything within the OR is checking if the HoH has any DEPENDENTS (excluding self)
+  #   that would make the HoH eligible to receive notifications for them.
+  scope :better_reminder_eligible, lambda {
+    joins(:dependents)
+      .monitoring_open
+      .within_preferred_contact_time
+      .has_not_reported_recently_enough_for_reminder_exlusion
+      .should_be_monitoring
+      .has_usable_preferred_contact_method
+      .where('patients.id = patients.responder_id')
+      .where(pause_notifications: false)
+      .where('patients.last_assessment_reminder_sent <= ? OR patients.last_assessment_reminder_sent IS NULL', 12.hours.ago)
```

Comment:
@holmesie Can we briefly revisit the 12 hours check here? I'm still not entirely convinced it's the best way to check this. Could this also be dependent on the `reporting_period_minutes` instead for consistency?

Reply:
I think if we keep the 12 hours then patients would get a reminder every day of the reporting period that they do not report. If we make it rely on the reporting period minutes (and have a reporting period > 1 day), then patients would only get a reminder once within a reporting period.

Reply:
I guess this is a very similar scenario to the one you diagramed in one of the above threads, and I would expect we'd want the latter, that way it's consistent with how we check the latest assessment submitted in `has_not_reported_recently`. The 12 hours feels somewhat arbitrary.

Reply:
Outside of this PR I believe we decided we'd be fine moving away from the 12 hours if we handle this the same way we handle `latest_assessment_at` in the new `has_not_reported_recently` scope here which uses `beginning_of_day`